lib/mcp/tool/input_schema.rb

```diff
@@ -15,8 +15,13 @@ def initialize(properties: {}, required: [])
         validate_schema!
       end
 
+      def ==(other)
+        other.is_a?(InputSchema) && other.properties == properties && other.required == required
```

Comment:
Can you refactor to the following?

```suggestion
        other.is_a?(InputSchema) && properties == other.properties && required == other.required
```